composer.json

```diff
@@ -5,9 +5,9 @@
     "license": "MIT",
     "require": {
         "loophp/phptree": "^2.5",
-        "illuminate/database": "~5.8.0|^6.0|^7.0",
-        "illuminate/events": "~5.8.0|^6.0|^7.0",
-        "illuminate/support": "~5.8.0|^6.0|^7.0"
+        "illuminate/database": "~5.8.0|^6.0|^7.0|^8.0",
```

Comment:
Following my comment about the API change, this PR can be changed with only Laravel 8 in mind 🤔